src/features/dashboard/components/ProfileCard/ProfileCard.module.scss

```
@@ -15,22 +15,25 @@
 .profileImage {
   width: 102px;
   height: 102px;
-  background: color.$white;
+  background: color.$gray-300;
   @include radius.card-radius;
   overflow: hidden;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)